docs/src/rules/no-useless-assignment.md

```diff
@@ -22,6 +19,10 @@ further_reading:
 "Dead stores" waste processing and memory, so it is better to remove unnecessary assignments to variables.
 
 Also, if the author intended the variable to be used, there is likely a mistake around the dead store.
+For example,
```

Comment:
Sorry, I don't think I was clear. I'm looking for a code example in this section, something like:

```js
let id = "x1234";    // this is a "dead store" - this value is never read

id = generateId();

doSomethingWith(id);
```

That way, we can explain that `"x1234"` is a value that can't possibly be used, and then we say this type of error can occur while refactoring.